src/filters/madgwickahrs.cpp

```diff
@@ -108,24 +120,24 @@ namespace dynamicgraph
       if(!m_initSucceeded)
         {
           SEND_WARNING_STREAM_MSG
-	    ("Cannot compute signal imu_quat before initialization!");
+            ("Cannot compute signal imu_quat before initialization!");
           return s;
         }
       const dynamicgraph::Vector& accelerometer = m_accelerometerSIN(iter);
       const dynamicgraph::Vector& gyroscope = m_gyroscopeSIN(iter);
 
       getProfiler().start(PROFILE_MADGWICKAHRS_COMPUTATION);
       {
-	// Update state with new measurment
-	madgwickAHRSupdateIMU
-	  (     gyroscope(0),     gyroscope(1),     gyroscope(2),
-		accelerometer(0), accelerometer(1), accelerometer(2));
-	if(s.size()!=4)
-	  s.resize(4);
-	s(0) = m_q0;
-	s(1) = m_q1;
-	s(2) = m_q2;
-	s(3) = m_q3;
+        // Update state with new measurment
+        madgwickAHRSupdateIMU
+          (     gyroscope(0),     gyroscope(1),     gyroscope(2),
+                accelerometer(0), accelerometer(1), accelerometer(2));
```

Comment:
Could you indent the code following the convention ?

Reply:
Will do